lib/zepto_client/api.rb

```diff
@@ -16,9 +16,9 @@ class Api
 
     attr_accessor :api_key, :base_url
 
-    def initialize(api_key)
+    def initialize(api_key, base_url)
```

Comment:
Why are we removing this? Check that it is in the Rails app that uses the gem, or put it back here. This seems more like a setting we need to pass in, not go by ENV.

So, init with api key and boolean for sandbox/prod

def initialize(api_key: , sandbox: true)

Reply:
Agreed, should have been an option to pass in rather than rely on environment so testing can happen in prod